Source/WebCore/editing/Editor.cpp

```diff
@@ -4229,11 +4229,11 @@ static Vector<TextList> editableTextListsAtPositionInDescendingOrder(const Posit
     textLists.reserveInitialCapacity(enclosingLists.size());
     for (auto iterator = enclosingLists.rbegin(); iterator != enclosingLists.rend(); ++iterator) {
         auto& list = iterator->get();
-        bool ordered = is<HTMLOListElement>(list);
+        auto* orderedList = dynamicDowncast<HTMLOListElement>(list);
         if (!list.renderer())
             continue;
         auto style = list.renderer()->style().listStyleType();
-        textLists.append({ style, ordered ? downcast<HTMLOListElement>(list).start() : 1, ordered });
+        textLists.append({ style, orderedList ? orderedList->start() : 1, (bool)orderedList });
```

Comment:
Best to avoid C-type casts. Either `static_cast<bool>(orderedList)` or `!!orderedList` would be better.